app/views/admin/_admin_nav.html.erb

```diff
@@ -1 +1,4 @@
-<h1 style="float:left;">Admin Dashboard</h1>
\ No newline at end of file
+<h1 style="float:left; margin:20px;">Admin Dashboard</h1>
+<h3 style="float:right; background-color:lightgrey; margin:20px;"><%= link_to "Dashboard", admin_path %> 
+<%= link_to "Merchants", admin_merchants_path %>   
+<%= link_to "Invoices", admin_invoices_path %></h3>
```

Comment:
I like that you split this into 3 lines for readibility instead of that one long line